lib/media/track/localvideotrack.js

```diff
@@ -183,8 +237,17 @@ class LocalVideoTrack extends LocalMediaVideoTrack {
       this._workaroundSilentLocalVideoCleanup();
       this._workaroundSilentLocalVideoCleanup = null;
     }
+
+    const processor = this.processor;
+    if (processor) {
+      this.removeProcessor(processor);
+    }
     const promise = super.restart.apply(this, arguments);
 
+    if (processor) {
+      promise.then(() => this.addProcessor(processor));
+    }
+
```

Comment:
This means that while restart is under process, `LocalVideoTrack.processor` and `LocalVideoTrack.processedTrack` is null. Instead of removing and adding the processor, can we stop and start capture?

Reply:
Document this behavior. Cut a ticket to change this behavior later, maybe during beta.